web/src/components/ApplicationCard/ApplicationCard.module.css

```
@@ -1,141 +1,142 @@
-.cardRead {
-  opacity: 0.7;
-}
-
+/* ---------- Status stamp: outlined, uppercase, icon + label always ---------- */
 .statusBadge {
   display: inline-flex;
   align-items: center;
   gap: var(--tc-space-xs);
   padding: var(--tc-space-xs) var(--tc-space-sm);
-  border-radius: var(--tc-radius-full);
+  border: 1.5px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)